src/Layers/FeatureLayer/FeatureLayer.js

```diff
@@ -89,8 +98,37 @@ EsriLeaflet.Layers.FeatureLayer = EsriLeaflet.Layers.FeatureManager.extend({
         this._map.addLayer(layer);
       }
 
+      // polylines and polygons
       if (layer && layer.setLatLngs) {
-        this._updateLayerGeometry(layer, geojson);
+        this._updateLayer(layer, geojson);
+        // update custom symbology, if necessary
+        this.resetStyle(geojson.id);
+      }
+
+      // points
+      if (layer && layer.setLatLng) {
+        this._updateLayer(layer, geojson);
+        // L.marker check
+        if (layer && layer.setIcon) {
+          // update custom symbology, if necessary
+          if (this.options.pointToLayer){
+            var getIcon = this.options.pointToLayer(geojson, L.latLng(geojson.geometry.coordinates[1], geojson.geometry.coordinates[0]));
+            var updatedIcon = getIcon.options.icon;
+            layer.setIcon(updatedIcon);
+          }
+        }
+        // L.circleMarker check
+        if (layer && layer.setStyle) {
+          if (this.options.pointToLayer){
+            var getStyle = this.options.pointToLayer(geojson, L.latLng(geojson.geometry.coordinates[1], geojson.geometry.coordinates[0]));
+            var updatedStyle = getStyle.options;
+            this.setFeatureStyle(geojson.id, updatedStyle);
+          }
+          else {
+            this.resetStyle(geojson.id);
```

Comment:
Same goes for here. I realize that is so we can catch updated rendering based on attributes but it also seems like it could be unwanted behavior.